### DISCUSSION TITLE
Access problem

### DISCUSSION THREAD

**Discussion by oeilneuf74** [iterate-ch/cyberduck#15345] [Q&A]
Hi!
i've been using cyberduck for several years without any problems. but for the past few days, i've been unable to open my bookmarks and thus access my files.

When I click on a bookmark, it takes a long time and I have this error message:

Connect timed out. The connection attempt timed out. The server may be down, or your network may not be properly configured.

Could you help me to solve that problem?
Thanks a lot.
Gaëlle

**Comment by tatze96** [comment]
Connection timeouts typically occur if the server doesn't answer in time on Cyberduck requests.
Please use either or both of the options below to address this issue:
1. Increase the set *Timeout for opening connections* within the Cyberduck preferences *Connection* tab.
2. Reach out to your server administrator or hosting provider to check if the server answers in time on requests.

If this doesn't resolve the connectivity issue, add information about the affected connection protocols and send us a [debug log](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level). If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15345 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
